pkg/pillar/hypervisor/kvm.go

```diff
@@ -730,6 +761,11 @@ func (ctx kvmContext) PCIRelease(long string) error {
 	return nil
 }
 
+func usbBusPort(USBAddr string) (string, string) {
+	ids := strings.SplitAfterN(USBAddr, ":", 2)
+	return ids[0], ids[1]
```

Comment:
If the user tells the controller a bogus string which doesn't follow the expect syntax this will panic. 
Better to check for bad format and report an error.

Reply:
Done. Those strings should come filtered, however.